openapi.yaml

```diff
@@ -9259,3 +9341,149 @@ components:
+        * `dep-success-count` - number of successful deployments in the time period.
+        * `dep-failure-count` - number of failed deployments in the time period.
+        * `dep-success-ratio` - ratio between successful and all deployments.
+        * `dep-size-prs` - average number of deployed pull requests.
+        * `dep-size-releases` - average number of deployed releases.
+        * `dep-size-lines` - average number of deployed lines of code (changed = insertions + removals).
+        * `dep-size-commits` - average number of deployed commits.
+        * `dep-prs-count` - number of pull requests deployed in the time period.
+        * `dep-releases-count` - number of releases deployed in the time period.
+        * `dep-lines-count` - number of deployed lines of code (changed = insertions + removals).
+        * `dep-commits-count` - number of deployed commits.
+        * `dep-repository-frequency` - average number of times a repository was deployed.
+        * `dep-jira-issues-count` - number of JIRA issues mapped to deployed pull requests.
+        * `dep-jira-bug-fixes-count` - number of deployed bug fixes.
+      enum:
+      - dep-count
+      - dep-duration
+      - dep-success-count
+      - dep-failure-count
+      - dep-success-ratio
+      - dep-size-prs
+      - dep-size-releases
+      - dep-size-lines
+      - dep-size-commits
+      - dep-prs-count
+      - dep-releases-count
+      - dep-lines-count
+      - dep-commits-count
+      - dep-repository-frequency
+      - dep-jira-issues-count
+      - dep-jira-bug-fixes-count
+      type: string
+    DeploymentMetricsRequest:
+      additionalProperties: false
+      description: Request body of `/metrics/deployments`, the deployments selector.
+      properties:
+        for:
+          description: List of repository groups for which to calculate the metrics.
+          items:
+            $ref: '#/components/schemas/RepositorySet'
+          type: array
+        with:
+          description: List of developer groups for which to calculate the metrics.
+          items:
+            $ref: '#/components/schemas/DeploymentWith'
+          type: array
+        metrics:
+          description: Requested metric identifiers.
+          items:
+            $ref: '#/components/schemas/DeploymentMetricID'
+          type: array
+        pr_labels_include:
+          description: At least one of these labels must be present in at least one
+            deployed PR.
+          items:
+            type: string
+          type: array
+        pr_labels_exclude:
+          description: None of these labels must be present in each deployed PR.
+          items:
+            type: string
+          type: array
+        jira:
+          $ref: '#/components/schemas/JIRAFilter'
+        with_labels:
+          description: Deployments should contain at least one of the specified label
```

Comment:
So these are the artificial labels we've added recently that users can notify us about right?

Reply:
Yep